### ISSUE TITLE
Inception resnet v2 padding

### ISSUE THREAD

**Issue by mazatov** [killthekitten/kaggle-carvana-2017#9]
Hello Nikolay, 

I saw that in your implementation of Inception Resnet v2 you had padding 'SAME' instead of 'VALID' padding in the original implementation from keras. I was wondering what was the reason for that? Is it to make Unet upsampling easier later on for segmentation? 

Is that the only change you did for the model?
